tests/Http/HttpTestingFileFactoryTest.php

```diff
@@ -95,9 +95,11 @@ public function testImageBmp()
 
         $imagePath = $image->getRealPath();
 
-        $this->assertSame('image/x-ms-bmp', mime_content_type($imagePath));
-
-        $this->assertSame('image/bmp', getimagesize($imagePath)['mime']);
+        if (version_compare(PHP_VERSION, '8.3.0-dev', '>=')) {
+            $this->assertSame('image/bmp', mime_content_type($imagePath));
+        } else {
+            $this->assertSame('image/x-ms-bmp', mime_content_type($imagePath));
+        }
```

Comment:
This seems like a behavioral change in PHP 8.3, I'm not sure if this is something the core team is aware of?